spec/lib/email/receiver_spec.rb

```diff
@@ -1381,6 +1380,23 @@ def reply_as_group_user
         expect(reply_post.reply_to_post_number).to eq(group_post.post_number)
       end
 
+      it "handles multiple message IDs in the in_reply_to header by only using the first one" do
+        email_log, group_post = reply_as_group_user
+
+        reply_email = email(:email_to_group_email_username_3)
+        reply_email.gsub!(
+          "MESSAGE_ID_REPLY_TO",
+          "<#{email_log.message_id}> <test/message/id@discourse.com>",
+        )
+        expect do Email::Receiver.new(reply_email).process! end.to not_change {
```

Comment:
god this formating is ugly what happened here

Reply:
This is the way syntax_tree wants it :shrug: :grimacing:

Reply:
iirc it tries to preserve `{ }`/`do end` it finds, so you could be able to at least change it to:

```suggestion
        expect { Email::Receiver.new(reply_email).process! }.to not_change {
```

🤞

Reply:
Ah ok will fix